src/components/Sponsors.tsx

```diff
@@ -0,0 +1,92 @@
+import { SPONSORS, type Sponsor, type SponsorType } from '@/data/sponsors';
+import Image from 'next/image';
+import FancyRectangle from './FancyRectangle';
+
+type SponsorCardProps = Sponsor & { reverse?: boolean };
+function SponsorCard({ image, name, description, website, type, reverse }: SponsorCardProps) {
+    return (
+        <div
+            className={`flex flex-col items-stretch gap-5 ${reverse ? 'md:flex-row-reverse' : 'md:flex-row'}`}
+        >
+            <div className="flex h-64 shrink-0 items-center justify-center bg-white md:h-auto md:w-64">
+                <Image
+                    src={`/images/sponsors/${image}`}
+                    alt={`${name} Logo`}
+                    width={200}
+                    height={200}
+                />
+            </div>
```

Comment:
I feel like the logo container might benefit from using `FancyRectangle` to add some colour to it.

Reply:
no, I'd like to follow the original design. Ask yuhan for advices?